src/widgets/notebook/NotebookEditor.tsx

```diff
@@ -93,71 +93,72 @@ export function Editor({ widget }: { widget: INotebookWidget }) {
           {isEditing ? <IconEditOff size={20} /> : <IconEdit size={20} />}
         </ActionIcon>
       )}
-      <RichTextEditor
-        p={0}
-        mt={0}
-        editor={editor}
-        styles={(theme) => ({
-          root: {
-            '& .ProseMirror': {
-              padding: '0  !important',
+      <ScrollArea h="100%" sx={{ borderRadius: '0.5rem' }}>
```

Comment:
The scroll should be on the content not the top of the widget including the bar.